relatio/embeddings.py

```diff
@@ -213,6 +219,27 @@ def get_vector(self, phrase: str) -> np.ndarray:
         return self._get_default_vector(phrase)
 
 
+class MultilingualBERTEmbeddings(EmbeddingsBase):
+    """
+    path = sentence-transformers/distiluse-base-multilingual-cased-v2
+    model = Embeddings("multilingual_BERT", path)
+    """
+
+    def __init__(
+        self, path: str = "sentence-transformers/distiluse-base-multilingual-cased-v2"
+    ) -> None:
+        try:
+            from sentence_transformers import SentenceTransformer
+        except ModuleNotFoundError:
```

Comment:
Could we directly install it for them? (just like the spacy modules)

Reply:
I've already added sentence-transformers in setup.cfg.
So, should this code be deleted?

Reply:
Should we also change codes in the same file below the same way as your comment?
(l.209-213)
`try:
            import tensorflow_hub as hub
        except ModuleNotFoundError:
            print("Please install tensorflow_hub package")
            raise
        self._embed = hub.load(path)`

(l.245-250)
`try:
            from gensim.models import Word2Vec

        except ModuleNotFoundError:
            print("Please install gensim package")
            raise`

(l.282-288)
`try:
            import gensim.downloader as api

        except ModuleNotFoundError:
            print("Please install gensim package")
            raise

        return api.load(model)`

Reply:
Yes, that would make sense.

Reply:
I would also check the package version (>= in the config file).

If it's not possible to pip for the users, I would change the Error Message to add also the package version restrictions.